qcodes/dataset/measurements.py

```diff
@@ -826,7 +832,7 @@ def unregister_parameter(self,
         self.parameters.pop(param)
         log.info(f'Removed {param} from Measurement.')
```

Comment:
@cgranade is it possible that you missed the `return self` here?

Reply:
My thought was that `unregister_parameter` is called in a fairly different way than the others, but I'm happy to add the `return self` here as well.

Reply:
Sorry @cgranade, there was a code fold that I overlooked so my comment ended up at the wrong line. I meant to refer to line 802, `register_custom_parameter`. Here the type `T` was added as a return type, but it is not returned.

Reply:
Ah, got it. Yes, that was a mistake I've since fixed.